plugins/package-managers/cocoapods/src/main/kotlin/CocoaPods.kt

```diff
@@ -229,7 +233,11 @@ private const val LOCKFILE_FILENAME = "Podfile.lock"
 
 private const val SCOPE_NAME = "dependencies"
 
-private fun getPackageReferences(podfileLock: File): Set<PackageReference> {
+private fun parseLockFile(
```

Comment:
I prefer if the functioned kept on returning the model of the parse lockfile instead of making it modify two given mutable data structures, because IMO that remains simpler as there are no side effects.

```
data class Lockfile(
     val packageReferences: Set<PackageReference>
     val externalSources: List<Package>
)
```